src/library/unfold_macros.cpp

```diff
@@ -119,8 +119,6 @@ class unfold_untrusted_macros_fn {
 };
 
 static bool contains_untrusted_macro(unsigned trust_lvl, expr const & e) {
-    if (trust_lvl > LEAN_BELIEVER_TRUST_LEVEL)
-        return false;
     return static_cast<bool>(find(e, [&](expr const & e, unsigned) {
```

Comment:
This version (`(unsigned trust_lvl, expr const & e)`) is only used when defining new definitions. The other one (`(unsigned trust_lvl, declaration const & d)`) is used during importing, and the optimization is fine. I think it is important to remove the check in this one though, because we currently pass the untrusted macros to the kernel without unfolding them, and the kernel throws an error.